gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
```

Comment:
My work depends on this gradle version, Without it I can not event compile a project, it was mentioned in the origin PR: https://github.com/web3j/web3j/pull/1860 

I can remove it and push commit, but it means pushing the code which you even can't compile on your machine. It is better to avoid it. Keeping in mind the nature of this changes, it is better to leave this PR open till your next major release when gradle version will be rised up.